inc/commonglpi.class.php

```diff
@@ -1193,6 +1193,11 @@ public static function isLayoutExcludedPage() {
    function display($options = []) {
       global $CFG_GLPI;
 
+      // Init active tab to main if not defined
+      if (!isset($_SESSION['glpi_tabs'][self::getType()])) {
+         Session::setActiveTab(self::getType(), self::getType().'$main');
```

Comment:
This may fail for objects that redefine tabs order in `defineDefaultObjectTabs`.